contracts/connectors/loantoken/LoanTokenLogicStandard.sol

```diff
@@ -500,56 +353,6 @@ contract LoanTokenLogicStandard is AdvancedToken, StorageExtension {
             .div(WEI_PRECISION);
     }
 
-    function getDepositAmountForBorrow(
```

Comment:
This is a breaking change for the UI. Relay this to Yuno please

Reply:
Kevin is aware of this. I am going to keep this in mind as well.